orchestra/communication/staffing.py

```diff
@@ -192,6 +192,7 @@ def get_available_requests(worker):
     remaining_requests = (
         StaffBotRequest.objects
         .filter(inquiries__communication_preference__worker=worker)
+        .exclude(status=StaffBotRequest.Status.COMPLETE.value)
```

Comment:
sanity-checking: is this function the one that's used for the available tasks page? have we manually tested the available tasks page to ensure that no existing requests disappear after this PR, and that requests do disappear after being assigned or completed?

Reply:
Yup.